src/quacc/recipes/espresso/dos.py

```diff
@@ -311,3 +358,88 @@ def projwfc_flow(
         "non_scf_job": non_scf_results,
         "projwfc_job": projwfc_results,
     }
+
+
+@flow
+def bands_flow(
+    atoms: Atoms,
+    job_decorators: dict[str, Callable | None] | None = None,
+    job_params: dict[str, Any] | None = None,
+) -> BandsSchema:
+    """
+    This function performs a bands calculation.
+
+    Consists of following jobs that can be modified:
+
+    1. pw.x static
+        - name: "static_job"
+        - job: [quacc.recipes.espresso.core.static_job][]
```

Comment:
I would suggest making this first step optional (e.g. `run_static: bool = True`) in case the user has already done a relax+static elsewhere and doesn't want to re-do that step. You can see an example of an optional step [here](https://github.com/Quantum-Accelerators/quacc/blob/2020cd6fb890df6fe2900ee4c9e6207e2091d9e7/src/quacc/recipes/vasp/slabs.py#L127).

That said, if my understanding is correct and it makes sense to merge `bands_job` and `bands_pp_job` into one step, then it does beg the question about whether a distinct flow is needed for this pattern. I would likely lean towards no if it's just static-->`bands_job`.